lib/textris/delivery/twilio.rb

```diff
@@ -2,14 +2,18 @@ module Textris
   module Delivery
     class Twilio < Textris::Delivery::Base
       def deliver(to)
-        options = {
-          :from => phone_with_plus(message.from_phone),
-          :to   => phone_with_plus(to),
-          :body => message.content
-        }
+        options = { to: phone_with_plus(to), body: message.content }
```

Comment:
One more nit-pick, sorry 🙏 Could you, please, keep old hash syntax to be consistent with app styling?